contrib/tele.centrum.v2/index.php

```diff
@@ -3,13 +3,19 @@
 error_reporting(E_ALL &~ E_NOTICE &~ E_DEPRECATED);
 
 require_once('..' . DIRECTORY_SEPARATOR . 'initLMS.php');
+require_once('lib' . DIRECTORY_SEPARATOR . 'definitions.php');
+$basedir=(__DIR__ . DIRECTORY_SEPARATOR . 'templates_c');
 
 $uid        = $_GET['id'];
 $phone      = $_GET['phone'];
 $agentnr    = $_GET['agentnr'];
 $ticket['phonetype'] = 'on';
 
-require_once('lib' . DIRECTORY_SEPARATOR . 'definitions.php');
+if (!is_dir($basedir)) {
+    die("Create Smarty Compile directory please run: mkdir " . $basedir);
```

Comment:
Poproszę o poprawienie angielskich komunikatów na bardziej poprawne językowo.

Reply:
Komunikaty już poprawiłem.

Reply:
To ja jeszcze zmieniłem error messages na taki sam styl "Prośliwy" ;-)